doc/examples/viz_advanced.py

```diff
@@ -0,0 +1,187 @@
+"""
+==================================
+Advanced interactive visualization
+==================================
+
+In DIPY we created a thin interface to access many of the capabilities
+available in the Visualization Toolkit framework (VTK) but tailored to the
+needs of structural and diffusion imaging. Initially the 3D visualization
+module was named ``fvtk``, meaning functions using vtk. This is still available
+for backwards compatibility but now there is a more comprehensive way to access
+the main functions using the following modules.
+"""
+
+import numpy as np
+from dipy.viz import actor, window, widget
+
+"""
+In ``window`` we have all the objects that connect what needs to be rendered
+to the display or the disk e.g. for saving screenshots. So, there you will find
```

Comment:
=> "(e.g., for saving screenhots)" 

Either way "e.g." (almost) always has a comma after it.